backend/package.json

```diff
@@ -1,55 +1,63 @@
 {
```

Comment:
New/modified scripts:
Prisma seed script so we can seed database with `npx prisma db seed`
genSchemas script for updating prisma schema. This generates the updated schemas from schema.prisma with zod-prisma-types and then we run prettier on the generated schemas to remove an unused import of the prisma client that is automatically created by zod-prisma-types. 

Changed the build and dev scripts:
- Include tsc-alias
- include /backend in the path for index.js, needed because of the use of lib adds another layer of directories

New dependencies: 
- multer
- @types/multer 
- zod
- zod-prisma-types

New dev dependency:
- tsc-alias // This is for correctly handling aliased paths (eg @lib, @prisma) when transpiling with tsc